src/pages/fieldManagement/settings/defaultFieldComponents/Editors.jsx

```diff
@@ -3,40 +3,38 @@ import { get } from 'lodash-es';
 
 import TreeEditor from './TreeEditor';
 import ConfigureDefaultField from './ConfigureDefaultField';
+import RelationshipEditor from './RelationshipEditor';
 
-export function RelationshipEditor() {
-  return null;
-}
-
-// export function RelationshipEditor({
-//   onClose,
-//   onSubmit,
-//   formSettings,
-//   setFormSettings,
-//   ...rest
-// }) {
-//   const speciesOptions = get(formSettings, 'relationships', []);
+export function RelationshipEditorWrapper({
+  onClose,
+  onSubmit,
+  formSettings,
+  setFormSettings,
+  children,
+  ...rest
+}) {
+  const transformedRelationshipOptions = get(
+    formSettings,
+    'relationships',
+    formSettings,
+  );
 
-//   return (
-//     <OptionEditor
-//       open
-//       onClose={onClose}
-//       onSubmit={onSubmit}
-//       schema={{
-//         labelId: 'EDIT_RELATIONSHIPS',
-//         descriptionId: 'EDIT_RELATIONSHIPS_DESCRIPTION',
-//       }}
-//       value={speciesOptions}
-//       onChange={newOptions => {
-//         setFormSettings({
-//           ...formSettings,
-//           relationships: newOptions,
-//         });
-//       }}
-//       {...rest}
-//     />
-//   );
-// }
+  return (
+    <ConfigureDefaultField onClose={onClose} onSubmit={onSubmit} open>
+      <RelationshipEditor
+        schema={{ labelId: 'RELATIONSHIPS' }}
+        value={transformedRelationshipOptions}
+        onChange={relationships => {
+          const formSettingObj = {};
+          formSettingObj['relationships'] = relationships;
+          setFormSettings(formSettingObj);
```

Comment:
setFormSettings({ relationships })

Reply:
This works because of 
```
const transformedRelationshipOptions = get(
    formSettings,
    'relationships',
    formSettings,
  );
```
, right? Otherwise, I don't understand why this worked.